### PR TITLE
Traffic creator fix

### PR DESCRIPTION
**Pull request by MTBC-Go-MF56**

### CODE REVIEW

File: map/map_creator.py [hunk 1]
```diff
@@ -50,8 +50,9 @@ def create_roads(roads):
             lanes = MapCreator.create_lanes(data["lanes"], data["road_type"], data["starting_pos"],
                                             data["length"], data["bearing"])
             data = (roads[str(i)])
+            road_width = len(lanes)*lanes[0].width
```
Comment (Nabeel07925) [inline map/map_creator.py:53]: road_width should come from configs, if it's defined there.

File: simulation/vehicle/traffic_creator.py [hunk 2]
```diff
@@ -78,12 +79,28 @@ def set_position(v, map1, taken):
...
         v.y = lane_points[xy_id][1]
-
         v.front_point = (v.x, upper_limit)
         v.back_point = (v.x, lower_limit)
-
         return v
 
+    @staticmethod
+    def __is_tuple_valid(tup, taken, lane_points, xy_id, car_length, map1, road_idx, lane_idx):
+        _taken = taken.copy()
```
Comment (Nabeel07925) [inline simulation/vehicle/traffic_creator.py:99]: a small docstring will really add a feather in the cap.

File: map/map_creator.py (hunk 1 above)
Comment (MTBC-Go-MF56) [reply map/map_creator.py:53]: road_width is not in config file, it is calculated from the lanes' width

**Review by Nabeel07925 — CHANGES_REQUESTED** [review]
Please update according to comments

**Review by Nabeel07925 — APPROVED** [review]

**Pull request merged**